components/console-backend-service/internal/domain/assetstore/dep.go

```diff
@@ -0,0 +1,60 @@
+package assetstore
```

Comment:
The file should contain only:
```go
//go:generate mockery -name=fileSvc -output=automock -outpkg=automock -case=underscore
type fileSvc interface {
	FilterByExtensions(statusRef *v1alpha2.AssetStatusRef, filterExtensions []string) ([]*File, error)
}

//go:generate mockery -name=gqlFileConverter -output=automock -outpkg=automock -case=underscore
type gqlFileConverter interface {
	ToGQL(file *File) (*gqlschema.File, error)
	ToGQLs(files []*File) ([]gqlschema.File, error)
}

type notifier interface {
	AddListener(observer resource.Listener)
	DeleteListener(observer resource.Listener)
}
```

Why? Because `dep.go` contains only shared interfaces between files. From what I've checked interfaces like `clusterAssetSvc` are used in single files. So move them to the files where they are used.